application/src/main/java/com/sanction/lightning/authentication/FacebookAuth.java

```diff
@@ -0,0 +1,26 @@
+package com.sanction.lightning.authentication;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+public class FacebookAuth {
```

Comment:
I don't like this class name so much.. but I don't have any ideas of what else to call it, maybe you have some ideas?